openmole/bin/org.openmole.site/jvm/src/main/scalatex/openmole/documentation/language/Hook.scalatex

```diff
@@ -197,7 +197,24 @@ You can think of the @code{DisplayHook} as an OpenMOLE equivalent to Scala's @co
   val h = DisplayHook("The value of i is ${i}.")
 """, name = "display hook")
 
+@h2{Variable restriction}
+You may want to restrict the hooked variables to a subset, like a variable filter.
+You can use the following notation in the hook function {@code{hook(yourHookHere, values = Seq(i,j))}}
+ 
+@br
+
+@hl.openmole("""
+  val i = Val[Int]
+  val j = Val[Int]
+  val k = Var[Double]
 
+  val task = ATask() set (
```

Comment:
ATask() => EmptyTask()

Reply:
Well, it was pseudo-code, but I forgot it was compiled.

Reply:
Could you fix the task type as well ?